### PR TITLE
bpo-34037: Add Python API whatsnew for loop.shutdown_default_executor()

### PR DESCRIPTION
**Pull request by aeros**
https://bugs.python.org/issue34037

### CODE REVIEW

File: Doc/whatsnew/3.9.rst
Comment (vstinner) [inline Doc/whatsnew/3.9.rst]:
```suggestion
* :meth:`asyncio.loop.shutdown_default_executor` has been added to
```

**Review by vstinner — APPROVED** [review]
LGTM, apart the typo ;-)

File: Doc/whatsnew/3.9.rst
Comment (remilapeyre) [inline Doc/whatsnew/3.9.rst:830]:
```suggestion
* :meth:`asyncio.loop.shutdown_default_executor` has been added to
```

Comment (aeros) [reply Doc/whatsnew/3.9.rst]: Oops, I have no idea how that happened. Thanks for mentioning it. :-)

**Comment by aeros** [comment]
Thanks for the review @vstinner. I'll merge it after the CI finish.

**Comment by aeros** [comment]
I wanted to try merging via the GitHub "Squash and Merge" instead of with automerge this time. I told myself just a minute ago "remember to change it to `GH-`", and I still completely forgot, haha.

But, I suppose that's somewhat inevitable as a new core dev, seeing as experienced core devs still manage to forget this step on occasion. :-)

**Pull request merged**
